cmd/podman/ps.go

```diff
@@ -405,66 +480,64 @@ func (p *psTemplateParams) headerMap() map[string]string {
 	return values
 }
 
+func sortPsOutput(sortBy string, psOutput psSorted) psSorted {
+	switch sortBy {
+	case "id":
+		sort.Sort(psSortedId{psOutput})
+	case "image":
+		sort.Sort(psSortedImage{psOutput})
+	case "command":
+		sort.Sort(psSortedCommand{psOutput})
+	case "runningfor":
+		sort.Sort(psSortedRunningFor{psOutput})
+	case "status":
+		sort.Sort(psSortedStatus{psOutput})
+	case "size":
+		sort.Sort(psSortedSize{psOutput})
+	case "names":
+		sort.Sort(psSortedNames{psOutput})
+	default:
```

Comment:
Are we sure we want this as the default? Would a case for "" being `psSortedCreated` and then default returning an error because the user probably entered an invalid sort be better?